model/src/pyrenew/datasets/data-raw/infection_admission_interval.py

```diff
@@ -0,0 +1,27 @@
+# Dataset from:
+# https://raw.githubusercontent.com/CDCgov/wastewater-informed-covid-forecasting/0962c5d1652787479ac72caebf076ab55fe4e10c/input/saved_pmfs/inf_to_hosp.csv
+import os
+
+import polars as pl
+
+# Read CSV file from URL
+url = "https://raw.githubusercontent.com/CDCgov/wastewater-informed-covid-forecasting/0962c5d1652787479ac72caebf076ab55fe4e10c/input/saved_pmfs/inf_to_hosp.csv"
+infection_admission_interval = pl.read_csv(url)
+
+# Building path to save the file
+path = os.path.join(
+    "src",
+    "pyrenew",
+    "datasets",
+    "infection_admission_interval.tsv",
+)
+
+assert os.path.exists(os.path.dirname(path))
```

Comment:
Should create if not exist. Can use this, which is equivalent to `mkdir -p` at the command line
```suggestion
os.makedirs(os.path.dirname(path), exist_ok=True)
```